Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -139,7 +139,7 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
                 continue;
             }
 
-            if ($token->equals(',')) {
+            if ($token->equals(',') && !in_array($tokens[$index - 1]->getContent(), array(')', ']'), true)) {
```

Comment:
`Tokens` or `Token[]` or array of token prototype, where each prototype may be as well array like string.
eg:
```php
array(
    new Token(...),
    array(T_STRING, 'foo'),
    '('
)
```